packages/@lwc/integration-karma/test/rendering/style-class-whitespace/x/component/component.html

```diff
@@ -0,0 +1,28 @@
+<template>
+    <div style="color: red !important "></div>
+    <div style="color: red !important ;"></div>
+    <div style="color: red  !important "></div>
+    <div style="color: red  !important ;"></div>
+    <div style="color: red ! important;"></div>
+    <div style="color: red ! important"></div>
+    <div style="color: red !IMPORTANT "></div>
+    <div style="color: red !IMPORTANT ;"></div>
+    <div style="color: red  !IMPORTANT "></div>
+    <div style="color: red  !IMPORTANT ;"></div>
+    <div style="color: red ! IMPORTANT;"></div>
+    <div style="color: red ! IMPORTANT"></div>
+    <div style="  color  :  red  !  IMPORTANT  ;  "></div>
+    <div style="  color  :  red  !  IMPORTAnt  ;  "></div>
+    <div style="  color  :  red    ;  "></div>
+    <div style="color: red; background-color: aqua  "></div>
+    <div style="color: red ; background-color: aqua; "></div>
+    <div style="  --its-a-tab:	red    ;  "></div>
+    <div style="  --its-a-tab-and-a-space:	 red    ;  "></div>
+    <div class=" boo "></div>
+    <div class="	boo"></div>
+    <div class=" foo bar   "></div>
+    <div class="foo   bar"></div>
+    <div class="	foo  bar "></div>
+    <div class=" foo	bar "></div>
+    <div class=" foo bar	"></div>
+</template>
```

Comment:
Adding this same template as a fixture test for `@lwc/engine-server` would be a nice touch.